api/pkg/runner/stub_windows.go

```diff
@@ -103,11 +104,26 @@ func (d *DiffusersRuntime) Status(_ context.Context) string {
 	panic("unimplemented")
 }
 
-type OllamaRuntime struct{}
+type OllamaRuntime struct {
```

Comment:
Blergh making ollama changes again.

Also what's with the interfaces?

I haven't tested that ollama even works on windows.

Probably revert this.

Reply:
Ollama isn't meant to work on windows. This is just a stub so we can still compile the bloody CLI :)